From 991f2d696af5fcac3a419ae64623e9703ea4bb2f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 20 Jul 2016 23:46:23 +0200
Subject: [PATCH] cogl-gles2: Don't leak gles2 types into mutter

On i686, the GLsizeiptr typedef in cogl-gles2.h conflicts with the
system GL headers, so make sure we don't include both.

https://bugzilla.gnome.org/show_bug.cgi?id=769014
---
 cogl/cogl/cogl-gles2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cogl/cogl/cogl-gles2.h b/cogl/cogl/cogl-gles2.h
index a90d0ce..87a06f1 100644
--- a/cogl/cogl/cogl-gles2.h
+++ b/cogl/cogl/cogl-gles2.h
@@ -65,7 +65,7 @@
  * conflicts we only expose the minimal set of GLES 2.0 types and enums
  * publicly.
  */
-#ifdef COGL_COMPILATION
+#if defined(COGL_COMPILATION) || defined(COGL_ENABLE_MUTTER_API)
 #include "cogl-gl-header.h"
 #else
 #include <cogl/cogl-gles2-types.h>
-- 
2.7.4

